administrator/components/com_content/Model/FeaturedModel.php

```diff
@@ -88,7 +90,7 @@ protected function getListQuery()
 			$this->getState(
 				'list.select',
 				'a.id, a.title, a.alias, a.checked_out, a.checked_out_time, a.catid, a.state, a.access, a.created, a.hits,' .
-					'a.created_by, a.featured, a.language, a.created_by_alias, a.publish_up, a.publish_down, a.note'
+				'a.created_by, a.featured, a.featured, fp.featured_up, fp.featured_down, a.language, a.created_by_alias, a.publish_up, a.publish_down, a.note'
```

Comment:
```suggestion
				'a.created_by, a.featured, fp.featured_up, fp.featured_down, a.language, a.created_by_alias, a.publish_up, a.publish_down, a.note'
```
One time `a.featured` should be enough ;-)